cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -438,6 +452,7 @@ cert-manager-openshift-routes-75b6bb44cd-f8kd5   1/1     Running   0          6s
 cert-manager-webhook-8498785dd9-bvfdf            1/1     Running   0          4h41m
 ----
 
+[id="cloud-experts-dynamic-certificate-custom-domain-config-deploy-sample-app"]
 == Deploying a sample application
 
```

Comment:
Some small intro here like "Now that dynamic certificates are configured, we can deploy a sample application to confirm that certificates are provisioned and trusted when we expose a new route" would be useful I think?

Reply:
Yes-thanks!